### PR TITLE
Adding a cleanup method to remove all artifacts after a test case

### PR DESCRIPTION
**Pull request by ramindu90**

### CODE REVIEW

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java [hunk 1]
```diff
@@ -103,13 +111,13 @@ protected String getXMLArtifactConfiguration(String testCaseFolderName, String c
...
      * @param testCaseFolderName testCaseFolderName Name of the folder created under /artifacts/CEP for the particular test case.
-     * @param configFileName Name of the JSON config-file created under above folder.
+     * @param configFileName     Name of the JSON config-file created under above folder.
      * @return The above JSON-configuration, as a string
      * @throws Exception
      */
     protected String getJSONArtifactConfiguration(String testCaseFolderName, String configFileName)
             throws Exception {
-        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/"+testCaseFolderName+"/"+configFileName;
+        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/" + testCaseFolderName + "/" + configFileName;
```
Comment (keizer619) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: Use File.separator instead /

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java [hunk 2]
```diff
@@ -89,13 +97,13 @@ protected void gracefullyRestartServer() throws Exception {
...
      * @param testCaseFolderName Name of the folder created under /artifacts/CEP for the particular test case.
-     * @param configFileName Name of the XML config-file created under above folder.
+     * @param configFileName     Name of the XML config-file created under above folder.
      * @return The above XML-configuration, as a string
      * @throws Exception
      */
     protected String getXMLArtifactConfiguration(String testCaseFolderName, String configFileName)
             throws Exception {
-        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/"+testCaseFolderName+"/"+configFileName;
+        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/" + testCaseFolderName + "/" + configFileName;
```
Comment (keizer619) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: @ramindu90 
Use File.separator instead "/"

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java [hunk 3]
```diff
@@ -117,16 +125,17 @@ protected String getJSONArtifactConfiguration(String testCaseFolderName, String
...
+     *
+     * @param testCaseFolderName    testCaseFolderName Name of the folder created under /artifacts/CEP for the particular test case.
      * @param executionPlanFileName Execution plan file name, relative to the test artifacts folder.
      * @return execution plan as a string.
      * @throws Exception
      */
     protected String getExecutionPlanFromFile(String testCaseFolderName, String executionPlanFileName)
             throws Exception {
-        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/"+testCaseFolderName+"/"+executionPlanFileName;
+        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/" + testCaseFolderName + "/" + executionPlanFileName;
```
Comment (keizer619) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: @ramindu90 
Use File.separator instead "/"

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java (hunk 2 above)
Comment (ayash) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: And also as a practice don't put string literals in the code like this. Use the constants class to put these all literals there and reference them from here.

Comment (ramindu90) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: Thanks @keizer619, "/" has replaced by File.separator in the following as well.

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java (hunk 1 above)
Comment (ramindu90) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: Thanks @keizer619, "/"  already gets replaced by the File.separator

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java (hunk 3 above)
Comment (ramindu90) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java]: Thanks @keizer619, "/" already gets replaced by the File.separator

**Pull request merged**